package.json

```diff
@@ -63,6 +64,7 @@
     "classnames": "2.2.5",
     "color": "^3.1.2",
     "config": "1.28.1",
+    "dompurify": "^2.0.7",
```

Comment:
Make sure you also commit in the yarn lock file when you make changes to `package.json`

Reply:
done